skills_ml/algorithms/occupation_classifiers/test.py

```diff
@@ -0,0 +1,45 @@
+from skills_ml.algorithms.preprocessing import IterablePipeline
+from skills_ml.algorithms.occupation_classifiers.classifiers import CombinedClassifier
+from skills_ml.algorithms.string_cleaners import nlp
+from skills_ml.job_postings.common_schema import JobPostingGeneratorType
+
+from typing import List, Callable
+from functools import partial
+from itertools import tee
+
+class OccupationClassifierTester(object):
+    def __init__(self,
+            test_data_generator: JobPostingGeneratorType,
+            preprocessing: List[Callable],
+            classifier: CombinedClassifier):
+        self.test_data_generator = test_data_generator
+        self.classifier = classifier
+        self.target_variable = self.classifier.target_variable
```

Comment:
In the current implementation somebody could just change `self.target_variable` to be something else, without changing `self.classifier`. Would that break things, or at least lead to confusing results? If so, target_variable should be a property. If not, this is good.

Reply:
I'll make it as the property.